skbio/stats/composition.py

```diff
@@ -708,6 +715,10 @@ def ancom(table, grouping,
 
         `"reject"` indicates if feature is significantly different or not.
 
+    pd.DataFrame
```

Comment:
This seems to render ok: 

![screenshot 2016-06-08 13 14 38](https://cloud.githubusercontent.com/assets/192372/15909380/fa261c1a-2d7a-11e6-8214-35c50e727f7f.png)